src/librbd/operation/EnableFeaturesRequest.cc

```diff
@@ -189,7 +190,8 @@ Context *EnableFeaturesRequest<I>::handle_get_mirror_mode(int *result) {
     }
     if ((m_features & RBD_FEATURE_FAST_DIFF) != 0) {
       if ((m_new_features & RBD_FEATURE_OBJECT_MAP) == 0) {
-	lderr(cct) << "cannot enable fast diff" << dendl;
+	lderr(cct) << "cannot enable fast-diff. object-map must be"
+                      " enabled before enabling fast-diff" << dendl;
```

Comment:
Nit: move the space back to the end of the previous line's string. Comment applies below as well.